wikimedia/wikipedia-ios
# Prevent navigationBar from hiding when voiceOver is activated

**Phabricator:** https://phabricator.wikimedia.org/T286821

### Notes
*  This PR adds an observer to prevent the navigation bar in the explore tab from hiding, so it maintains the search bar, settings button, and notifications button always reachable to voiceOver users

### Test Steps
1. Run the app with the explore feed enabled
2. Enable voiceOver (from Siri or from settings)
3. Scroll up and down the explore feed, make sure the navigation bar and search bar do not hide
4. Turn off VoiceOver, and scroll up and down the explore feed. Make sure the current hiding behavior for the navigation bar still works well
